tests/tests.sh

```diff
@@ -17,8 +18,27 @@ docker build -t ${DOCKER_IMAGE_NAME} .
 readonly USER="${AWS_ACCESS_KEY_ID}"
 readonly PASS="${AWS_SECRET_ACCESS_KEY}"
 readonly WORKFLOW_PATH="${GH_ACTION_WORKFLOW_PATH}"
+readonly TIMEOUT="${ACTION_TIMEOUT}"
+
+# Trap ctrl+c to remove docker container and kill timeout script.
+trap ctrl_c INT
+function ctrl_c() {
```

Comment:
```suggestion
function ctrl_c {
```

Reply:
>The function keyword is extraneous when “()” is present after the function name, but enhances quick identification of functions.

https://google.github.io/styleguide/shellguide.html#function-names

As I understand, we should use or `function name {}` or `name() {}`

In all other funtions used `function name {}` style

Reply:
done